vagrant/roles/logs.collect/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Create log directory structure
+  file:
+    path: "{{ config.statedir }}/{{ inventory_hostname }}"
+    state: directory
+  delegate_to: localhost
+
+- name: Copy /var/log directory
+  synchronize:
+    src: /var/log
+    dest: "{{ config.statedir }}/{{ inventory_hostname }}"
+    mode: pull
+
+- name: Process backend specific tasks
+  include_role:
+    name: 'sit.{{ config.be.name }}'
+    tasks_from: logs/main.yml
```

Comment:
I would move this before the copy of /var/log so that the backend has a chance to stop things correctly (if necessary), do whatever final checks are necessary, and move things to /var/log instead of copying them itself to the host.

Another alternative is to pass a variable with the destination to use, but I think that just requiring backends to store any logs in /var/log is cleaner and more flexible.

Reply:
Done.